typescript/sdk/src/token/EvmERC20WarpModule.ts

```diff
@@ -202,6 +205,35 @@ export class EvmERC20WarpModule extends HyperlaneModule<
     return updateTransactions;
   }
 
+  /**
+   * Transfer ownership of an existing Warp route with a given config.
+   *
+   * @param actualConfig - The on-chain router configuration.
+   * @param expectedConfig - The expected token router configuration.
+   * @returns Ethereum transaction that need to be executed to update the owner.
+   */
+  async transferOwnership(
+    actualConfig: TokenRouterConfig,
+    expectedConfig: TokenRouterConfig,
+  ): Promise<AnnotatedEV5Transaction[]> {
```

Comment:
```suggestion
  transferOwnership(
    actualConfig: TokenRouterConfig,
    expectedConfig: TokenRouterConfig,
  ): AnnotatedEV5Transaction[] {
```
no need to await

please do the same for `updateRemoteRouters` above since it is synchronous

Reply:
Removed